### DISCUSSION TITLE
Is it possible to blacklist and hide a folder without whitelisting everything else?

### DISCUSSION THREAD

**Discussion by danielkrajnik** [netblue30/firejail#4821] [Q&A] [answered]
from https://github.com/netblue30/firejail/wiki/Creating-Profiles#common-mistakes

> Common mistakes
> blacklist PATH: PATH is still present, but not accessible (with whitelist they don't exist)

is it possible to specificy a directory that should be inaccessible and not even visible in a sandbox?

**Comment by rusty-snake** [comment, marked as answer]
With firejail: no

Technical: You must always whitelist (with a parent directory as base) unless you intercept syscalls (LSM).

**Reply by danielkrajnik** [reply]
thank you again, that's good to know. Interesting, haven't seen LSM before
